server/tests-py/queries/v1/metadata/setup.yaml

```diff
@@ -104,7 +104,7 @@ args:
 - type: run_sql
   args:
     sql: |
-      CREATE FUNCTION fetch_articles(search text, author_row author)
+      CREATE FUNCTION fetch_articles(search text, author_row author, session json)
```

Comment:
Any reason why this test case has to be modified? We can let it be so that it tests the `session_argument` not being specified?

Reply:
Ah, oops, it's because I first considered adding the test to `fetch_articles`, and then I changed my mind. I'll remove it again.

Reply:
You're right, I'll separate it.